### DISCUSSION TITLE
Repetitive mask with displayChar

### DISCUSSION THREAD

**Discussion by ghost** [uNmAnNeR/imaskjs#910] [Q&A]
I am trying to create a simple mask that will display all character as '*'

Couldn't figure it out. Furthest I have gotten is the repetitive mask part.
Appreciate any helps.
```
IMask(el, {
  mask: '+char',
  lazy: true,
  blocks: {
    char: {
      mask: /^.*$/,
      displayChar: '*',
    }
  }
})
```

**Comment by uNmAnNeR** [comment]
there is no repeat option available for the pattern mask. It must be fixed length.

**Reply by ghost** [reply]
@uNmAnNeR Thanks.

Will format/parse allow me to achieve this? Or is that not possible either?

**Reply by uNmAnNeR** [reply]
probably `validate` can help
